ROLES.md

```diff
@@ -164,6 +165,47 @@ a 180 day period, than that individual may lose membership. On-going contributio
 
 Members who frequently contribute code are expected to proactively perform code reviews for the area that they are active in.
 
+## Documentation Reviewer
+
+A **Documentation Reviewer** is trusted to only approve
+content that meets the acceptance criteria described in the
+[contribution guides](https://istio.io/about/contribute/).
+
+### Requirements
+
+To become a **Reviewer**, contributors must meet the following **requirements**:
+
+* Be a **Member** of the Istio community.
+* Perform 5 substantial contributions to the `istio.io` repo. Substantial
+  contributions include the following examples:
+
+    * New content
+    * Content reviews
+    * Content improvements
+
+* Demonstrate a solid commitment to documentation quality and use of our style
+  guide.
+* Be sponsored by a Istio Maintainer or WG Lead.
+
+### Responsibilities
+
+A **Reviewer** would have the following **responsibilities** in Istio:
```

Comment:
Remove this line here and under privileges, just state the responsibilities and privileges.